dotnet/src/Experimental/Process.LocalRuntime/LocalKernelProcessContext.cs

```diff
@@ -14,17 +13,22 @@ public sealed class LocalKernelProcessContext : KernelProcessContext, IDisposabl
     private readonly LocalProcess _localProcess;
     private readonly Kernel _kernel;
 
-    internal LocalKernelProcessContext(KernelProcess process, Kernel kernel)
+    internal LocalKernelProcessContext(KernelProcess process, Kernel kernel, ProcessEventProxy? filter = null)
```

Comment:
Would `eventProxy` be a better name for the parameter? Given how it is used if a developer is using named parameters it might be clearer.

Reply:
Yes, agree.  Updated